### PR TITLE
Consistent capitalization of the word 'ID' in comments of db_schema.xml files

### PR DESCRIPTION
**Pull request by psdevhh**
See the issue here:

https://github.com/magento/community-features/issues/198

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=24638) <br/>All committers have signed the CLA.

**Comment by psdevhh** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @psdevhh. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @psdevhh, here is your [new Magento instance](https://pr-24638.instances.magento-community.engineering).
Admin access: [https://pr-24638.instances.magento-community.engineering/admin](https://pr-24638.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @psdevhh,
Thank you for your contribution!

Could you sign CLA and associate commit author email with your Github Account? Once you'll do that - CLA will have Success status, now it's "Pending":
image


Could you review integration test failures and adjust them?

**Comment by drpayyne** [comment]
Just for [reference](https://softwareengineering.stackexchange.com/a/186286) and knowledge regarding this:
> Acronyms differ from abbreviations in that an abbreviation shortens a single word. For example, ID is an abbreviation for identifier. In general, library names should not use abbreviations. The two abbreviations that can be used in identifiers are ID and OK. In Pascal-cased identifiers they should appear as Id, and Ok. If used as the first word in a camel-cased identifier, they should appear as id and ok, respectively.

**Comment by psdevhh** [comment]
Interesting, thanks for the link @drpayyne 

In this case though, it's strictly about the comments in the db_schema.xml files, not any class names or so. These are written in plain English and as far as I know abbreviations like ID are usually capitalized in standard written English. 

The more glaring issue is, that the capitalization is completely inconsistent, with sometimes "ID", "Id", or "id" being used. Better to choose one and be consistent about it I think.

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi @psdevhh,
Could you review failing integration tests & fix them?

Thank you!

**Comment by psdevhh** [comment]
@ihor-sviziev Yes, I'm on it, still trying to get the integration tests running locally so I can see why they're failing. I'll update this as soon as I figured it out.

**Comment by psdevhh** [comment]
Ok, I reverted some changes that broke the `dev/tests/setup-integration/testsuite/Magento/Developer/Console/Command/SetupInstallTest.php` test. 

In

`dev/tests/setup-integration/testsuite/Magento/Developer/_files/SetupInstall/TestSetupDeclarationModule9/db_schema.xml`

and

`dev/tests/setup-integration/testsuite/Magento/Developer/_files/SetupUpgrade/TestSetupDeclarationModule9/db_schema_upgrade.xml`

changing the `Module8 Entity Id` & `Module8 Contact Id` into `Module8 Entity ID` & `Module8 Contact ID` breaks that test for some reason.

Unfortunately I couldn't figure out what's going on from reading the code alone and wasn't able to get the tests running locally yet either (seems to be a bigger endeavour than I expected), making it hard to debug, which is why I reverted those changes for now.

Apart from that, feel free to review, give feedback or even consider merging this. Thanks.

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5936](https://jira.corp.magento.com/browse/ENGCOM-5936) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by magento-engcom-team** [comment]
@psdevhh thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed
